pipelines/open-edx/mfe_build_and_deploy.yaml

```diff
@@ -0,0 +1,96 @@
+---
+resource_types:
+- name: rclone
+  type: docker-image
+  source:
+    repository: mitodl/concourse-rclone-resource
+    tag: latest
+
+resources:
+- name: mfe-app-(( mfe_path ))
+  type: git
+  source:
+    uri: (( mfe_repository ))
+    branch: (( open_edx_release_name ))
+- name: mfe-app-bucket
+  type: rclone
+  source:
+    config: |
+      [s3-remote]
+      type = s3
+      provider = AWS
+      env_auth = true
+      region = us-east-1
+
+jobs:
+- name: compile-and-deploy-mfe-(( mfe_path ))-to-(( open_edx_environment ))
+  plan:
+  - get: mfe-app-(( mfe_path ))
+    trigger: true
+  - task: compile-and-deploy-mfe
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: node
+          tag: (( mfe_node_version ))-bullseye-slim
+      inputs:
+      - name: mfe-app-(( mfe_path ))
+      outputs:
+      - name: mfe-app-(( mfe_path ))/dist
+      params:
+        ABOUT_US_URL:
+        ACCESS_TOKEN_COOKIE_NAME: (( open_edx_environment ))-edx-jwt-cookie-header-payload
+        BASE_URL: https://(( open_edx_lms_domain ))
+        CREDENTIALS_BASE_URL:
+        CSRF_TOKEN_API_PATH: /csrf/api/v1/token
+        Contact:
+        DISCOVERY_API_BASE_URL:
```

Comment:
We still haven't deployed this. Maybe it's useful, but I don't have a use case for it.